### PR TITLE
fix upload bar

### PR DESCRIPTION
**Pull request by songgao**
* don't be a hobbit in widget
* make broken pattern smooth
* use small pattern asset like the payment one now that `repeat` works
  for Android
* fix bug on mobile where upload bar only shows in mounted screens

### CODE REVIEW

**Comment by songgao** [comment]
sorry this one's still having issues. Will tag review again when it's ready.

**Comment by zanderz** [comment]
As per our chat, it's not working right for me

**Comment by songgao** [comment]
OK this is ready again!

**Comment by songgao** [comment]
bump!

**Comment by songgao** [comment]
bumpy bump ⪮ @keybase/react-hackers

**Comment by songgao** [comment]
⪮

**Comment by jzila** [comment]
@songgao I think you need to fix some conflicts. FWIW I've tested and it works well, although my filename is always getting cut off by an ellipsis.

**Comment by jzila** [comment]
image

**Comment by jzila** [comment]
(that's `random.bin`)

**Comment by songgao** [comment]
Ah; good catch!

File: shared/fs/folder/index.js [hunk 1]
```diff
@@ -47,18 +46,9 @@ class Files extends React.PureComponent<FolderProps> {
           )}
           <Footer />
         </Kb.Box2>
-      </Kb.Box2>
+      </Kb.BoxGrow>
```
Comment (songgao) [inline shared/fs/folder/index.js]: @chrisnojima  is this what you meant?

**Comment by jzila** [comment]
Tested and approved

**Pull request merged**